content/en/user-guide/aws/acm/index.md

```diff
@@ -41,25 +36,142 @@ This command will return the Amazon Resource Name (ARN) of the new certificate,
 }
 ```
 
+### List the certificates
+
 Use the [`ListCertificates` API](https://docs.aws.amazon.com/acm/latest/APIReference/API_ListCertificates.html) to list all the certificates. This command returns a list of the ARNs of all the certificates that have been requested or imported into ACM. Here's an example command:
 
 {{< command >}}
 $ awslocal acm list-certificates --max-items 10
 {{< /command >}}
 
+### Describe the certificate
+
 Use the [`DescribeCertificate` API](https://docs.aws.amazon.com/acm/latest/APIReference/API_DescribeCertificate.html) to view the details of a specific certificate. Provide the ARN of the certificate you want to view, and this command will return information about the certificate's status, domain name, and other attributes. Here's an example command:
 
 {{< command >}}
 $ awslocal acm describe-certificate --certificate-arn arn:aws:acm:<region>:account:certificate/<certificate_ID>
 {{< /command >}}
 
+### Delete the certificate
+
 Finally you can use the [`DeleteCertificate` API](https://docs.aws.amazon.com/acm/latest/APIReference/API_DeleteCertificate.html) to delete a certificate from ACM, by passing the ARN of the certificate you want to delete. Here's an example command:
 
 {{< command >}}
 $ awslocal acm delete-certificate --certificate-arn arn:aws:acm:<region>:account:certificate/<certificate_ID>
 {{< /command >}}
 
-For more comprehensive information on ACM, refer to the [AWS Certificate Manager](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html) documentation. You can use the `awslocal` CLI or any [integration](https://docs.localstack.cloud/user-guide/integrations/) to interact with ACM in LocalStack.
+## ACM Private Certificate Authority
+
+AWS Private Certificate Authority (ACM PCA) is a managed private Certificate Authority (CA) service that manages the lifecycle of your private certificates. ACM PCA extends ACM's certificate management capabilities to private certificates, enabling you to manage public and private certificates centrally.
```

Comment:
Please add a newline between the sentences here as well as for the other changed lines (according to our best practices).